### DISCUSSION TITLE
与女仆互动以及增加女仆喜欢的食物的建议

### DISCUSSION THREAD

**Discussion by 15so42** [TartaricAcid/TouhouLittleMaid#348] [Suggestions & Ideas]
1.在家里或者在刷怪塔下等待时希望能与女仆互动，比如给女仆一枝花能让女仆高兴并说谢谢并提升好感度，给女仆蛋糕能让女仆恢复血量并做出吃蛋糕的动画，以及摸头的动画什么的。还有女仆之间希望也能互动，我发现两个女仆有时会对视半天，但就做不出更多的互动了，我觉得要是让他们能互相嬉戏玩闹就好了，让游戏变得热闹起来。

2.我觉得女仆果然还是要有喜欢的食物才行，向玩家索取，得到了会更高兴，这样的女仆才会更加可爱嘛。就像女仆酱模组里的女仆不吃糖就会罢工一样，加入类似糖的食物的话也会让玩家更加的关心女仆，而且糖可以让女仆在战斗中快速回血，这样也算增强了女仆，现在的女仆我就只敢让他在主世界晃晃，地狱不敢去，更别说我的虚无世界的其他维度了。。

3.感谢作者大大给我这么可爱的女仆，请继续努力让女仆变得更可爱吧！

**Comment by 3w3wcom** [comment]
+1
